Description: Don't use pydata-sphinx-theme, as it is currently broken

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Bug-Debian: https://bugs.debian.org/1084781
Forwarded: not-needed

--- pandas-2.2.3+dfsg.orig/doc/source/conf.py
+++ pandas-2.2.3+dfsg/doc/source/conf.py
@@ -221,7 +221,7 @@ pygments_style = "sphinx"
 
 # The theme to use for HTML and HTML Help pages.  Major themes that come with
 # Sphinx are currently 'default' and 'sphinxdoc'.
-html_theme = "pydata_sphinx_theme"
+
 
 # The style sheet to use for HTML and HTML Help pages. A file of that name
 # must exist either in Sphinx' static/ path, or in one of the custom paths
@@ -240,29 +240,6 @@ else:
     # only keep major.minor version number to match versions.json
     switcher_version = ".".join(version.split(".")[:2])
 
-html_theme_options = {
-    "external_links": [],
-    "footer_start": ["pandas_footer", "sphinx-version"],
-    "github_url": "https://github.com/pandas-dev/pandas",
-    "twitter_url": "https://twitter.com/pandas_dev",
-    "logo": {"image_dark": "https://pandas.pydata.org/static/img/pandas_white.svg"},
-    "navbar_align": "left",
-    "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
-    "switcher": {
-        "json_url": "https://pandas.pydata.org/versions.json",
-        "version_match": switcher_version,
-    },
-    # This shows a warning for patch releases since the
-    # patch version doesn't compare as equal (e.g. 2.2.1 != 2.2.0 but it should be)
-    "show_version_warning_banner": False,
-    "icon_links": [
-        {
-            "name": "Mastodon",
-            "url": "https://fosstodon.org/@pandas_dev",
-            "icon": "fa-brands fa-mastodon",
-        },
-    ],
-}
 
 # Add any paths that contain custom themes here, relative to this directory.
 # html_theme_path = ["themes"]
